pkg/k8s/watchers/node.go

```diff
@@ -117,3 +135,109 @@ func (k *K8sWatcher) GetK8sNode(_ context.Context, nodeName string) (*v1.Node, e
 	}
 	return nodeInterface.(*v1.Node).DeepCopy(), nil
 }
+
+// CiliumNodeUpdater implements the subscriber.Node interface and is used
+// to keep CiliumNode objects in sync with the node ones.
+type CiliumNodeUpdater struct {
```

Comment:
```suggestion
// ciliumNodeUpdater implements the subscriber.Node interface and is used
// to keep CiliumNode objects in sync with the node ones.
type ciliumNodeUpdater struct {
```

Doesn't need to be exported.